vuu-ui/packages/vuu-shell/src/layout-management/SaveLayoutPanel.tsx

```diff
@@ -1,5 +1,123 @@
+  const [group, setGroup] = useState<string>("");
+  const [checkValues, setCheckValues] = useState<string[]>([]);
+  const [radioValue, setRadioValue] = useState<RadioValue>(radioValues[0]);
+
+  return (
+    <div className="panelContainer">
+      <div className="panelContent">
+        <div className="formContainer">
+          <FormField>
+            <FormFieldLabel>Group</FormFieldLabel>
+            <ComboBox
+              ListProps={{
+                style: {
+                  zIndex: 10000,
+                  border: "1px solid #777C94",
+                  borderRadius: 10,
+                  boxSizing: "border-box"
+                }
+              }}
+              source={groups}
+              allowFreeText={true}
+              InputProps={{
+                inputProps: {
+                  placeholder: "Select Group or Enter New Name",
+                  onChange: (event: ChangeEvent<HTMLInputElement>) => setGroup(event.target.value),
+                },
+              }}
+              width={120}
+              onSelectionChange={(_, value) => setGroup(value || "")}
+            />
+          </FormField>
+          <FormField>
+            <FormFieldLabel>Layout Name</FormFieldLabel>
+            <Input
+              inputProps={{ placeholder: "Enter Layout Name" }}
+              onChange={(event: ChangeEvent<HTMLInputElement>) => setLayoutName(event.target.value)}
+              value={layoutName}
+            />
+          </FormField>
+          <FormField>
+            <FormFieldLabel>Some Layout Setting</FormFieldLabel>
+            <div className="settingsGroup">
+              {checkboxValues.map((value, i) =>
+                <Checkbox
+                  key={i}
+                  className="setting"
+                  onToggle={() => setCheckValues((prev) => prev.includes(value) ? prev.filter(entry => entry !== value) : [...prev, value])}
+                  checked={checkValues.includes(value)}
+                  label={value}
+                />
+              )}
+            </div>
+          </FormField>
+          <FormField>
+            <FormFieldLabel>Some Layout Setting</FormFieldLabel>
+            <div className="settingsGroup">
+              {radioValues.map((value, i) =>
+                <RadioButton
+                  key={i}
+                  onClick={() => setRadioValue(value)}
+                  checked={radioValue === value}
+                  label={value}
+                  groupName="radioGroup"
+                />
+              )}
+            </div>
+          </FormField>
+        </div>
+        <div className="screenshotContainer">
+          <div className="screenshot" />
+        </div>
+      </div>
+      <div className="buttonsContainer">
+        <Button className="cancelButton" onClick={onCancel}>
+          Cancel
+        </Button>
+        <Button
+          className="saveButton"
+          onClick={() => onSave(layoutName, group, checkValues, radioValue || "")}
```

Comment:
This was necessary because we were defining the `RadioValue` type as optional. This shouldn't be the case though, so I've updated the type and removed the `|| ""` here.